containers/proxy-httpd-image/Dockerfile

```
@@ -9,40 +9,32 @@ ARG PXYRELEASENOTESPKG="release-notes-uyuni-proxy"
 
 ARG PRODUCT_REPO
 
-# Add distro and product repos
-COPY add_repos.sh /usr/bin
-RUN sh add_repos.sh ${PRODUCT_REPO}
-
 # Main packages
-COPY remove_unused.sh .
-RUN echo "rpm.install.excludedocs = yes" >>/etc/zypp/zypp.conf
-RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-licenses  \
-    spacewalk-proxy-broker \
-    spacewalk-proxy-redirect \
-    spacewalk-proxy-html \
-    ${PXYRELEASENOTESPKG} \
-    python3-rhnlib \
-    python3-PyYAML && \
-    sh remove_unused.sh
+RUN [ -n "$PRODUCT_REPO" ] && zypper addrepo $PRODUCT_REPO product; \
+    zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-licenses  \
+        spacewalk-proxy-broker \
+        spacewalk-proxy-redirect \
+        spacewalk-proxy-html \
+        ${PXYRELEASENOTESPKG} \
+        python3-rhnlib \
+        python3-PyYAML && \
+    sysconf_addword /etc/sysconfig/apache2 APACHE_MODULES headers && \
+    sysconf_addword /etc/sysconfig/apache2 APACHE_MODULES proxy_http && \
+    zypper --non-interactive clean --all && \
+    rm -rf /var/log/{alternatives.log,lastlog,tallylog,suseconnect.log,zypper.log,zypp/history,YaST2}
```

```suggestion
RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-licenses  \
        spacewalk-proxy-broker \
        spacewalk-proxy-redirect \
        spacewalk-proxy-html \
        ${PXYRELEASENOTESPKG} \
        python3-rhnlib \
        python3-PyYAML && \
    sysconf_addword /etc/sysconfig/apache2 APACHE_MODULES headers && \
    sysconf_addword /etc/sysconfig/apache2 APACHE_MODULES proxy_http && \
    zypper --non-interactive clean --all && \
    rm -rf /var/log/{alternatives.log,lastlog,tallylog,suseconnect.log,zypper.log,zypp/history,YaST2}
```
I would drop the repo add: this is only to build the container images locally without OBS… which we never do anymore: we always build in OBS branches or using `osc build`